relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -30,6 +30,12 @@ const MAX_SEGMENT_LENGTH: usize = 25;
 /// Some bundlers attach characters to the end of a filename, try to catch those.
 const MAX_EXTENSION_LENGTH: usize = 10;
 
+const REDIS_COMMAND_SINGLE_KEY: [&str; 13] = [
+    "GET", "SET", "DEL", "EXISTS", "HGET", "HSET", "HDEL", "HEXISTS", "HLEN", "APPEND", "EXISTS",
```

Comment:
There are a lot more commands than this in Redis. We should try to parse them in a more generic fashion.